web-frontend/src/pages/CreateAccount/StepOne.tsx

```diff
@@ -1,37 +1,84 @@
-import { Input } from '@massalabs/react-ui-kit/src/components/Input/Input';
-import { Stepper } from '@massalabs/react-ui-kit/src/components/Stepper/Stepper';
-import { Button } from '@massalabs/react-ui-kit/src/components/Button/Button';
+import { useState, useRef, SyntheticEvent } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Intl from '../../i18n/i18n';
+import { routeFor } from '../../utils';
+import { parseForm } from '../../utils/parseForm';
+
+import { Input, Stepper, Button } from '@massalabs/react-ui-kit';
 import { FiArrowRight } from 'react-icons/fi';
 import LandingPage from '../../layouts/LandingPage/LandingPage';
-import { routeFor } from '../../utils';
-import { Link } from 'react-router-dom';
-import Intl from '../../i18n/i18n';
+
+interface IErrorObject {
+  nickname: string;
+}
 
 export default function StepOne() {
+  const navigate = useNavigate();
+  const form = useRef(null);
+
+  const [error, setError] = useState<IErrorObject | null>(null);
+
+  function validate(e: SyntheticEvent) {
+    const formObject = parseForm(e);
+    const { nickname } = formObject;
+
+    if (!nickname || !nickname.length) {
```

Comment:
Non blocking: only one of them is not enough!?

Reply:
Done